portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase

```diff
@@ -440,6 +440,64 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-11509 | Test Summary: Check Document Rating show expected value"
+	@priority = "4"
+	test CheckDocumentsAndMediaRatingShowExpectedValue {
+		task ("Connect the DXP to AC") {
+			ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+		}
+
+		task ("View the documents and media page") {
+			ACUtils.navigateToSitePage(
+				actionType = "View DM",
+				documentTitleList = "DM AC Title",
+				pageName = "AC Page",
+				siteName = "Site Name");
+		}
+
+		task ("Choose a rating star for the DM") {
+			Click(locator1 = "Icon#INFO");
```

Comment:
Hey, I think there's a macro for this: `DMDocument.expandInfo();`

Reply:
Can you also change in this test too? `DocumentsAndMediaRatingsNotGettingNegativeWhenDeleteVote`